### PR TITLE
Enhance Qualimap BAMQC

### PR DESCRIPTION
**Pull request by MaxUlysse**
Following @parlundin request:
- Use more cpus in parrallel
- Add some extra flags
- Add one extra BAMQC process to run on the mapped bams

## PR checklist
 - [X] PR is made against `dev` branch
 - [X] This comment contains a description of changes (with reason)
 - [X] Ensure the test suite passes (`./scripts/test.sh -p docker -t ALL`).
 - [x] `CHANGELOG.md` is updated

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -174,6 +175,40 @@ if (params.verbose) mappedBam = mappedBam.view {
...
+  when: !params.noReports && !params.noBAMQC
+
+  script:
+  """
+  qualimap --java-mem-size=${task.memory.toGiga()}G \
+  bamqc \
+  -bam ${bam} \
+  --paint-chromosome-limits \
+  --genome-gc-distr HUMAN \
+  -nt 16 \
```
Comment (alneberg) [inline main.nf]: Hardcode nr of threads?

Comment (MaxUlysse) [reply main.nf]: :man_facepalming:

Comment (MaxUlysse) [reply main.nf]: It is now corrected...
Thanks for noticing that

**Comment by MaxUlysse** [comment]
Quick question, since we enhanced BAMQC, should we do the same for samtools stats?

**Pull request merged**